### DISCUSSION TITLE
Question about Forward Nodes

### DISCUSSION THREAD

**Discussion by Shorider** [Security-Onion-Solutions/securityonion#5843] [Unsupported Versions] [answered]
Hello,

I have been recently introduced to Security Onion and am actively trying to teach myself everything that I can to get caught up.  I am also fielding questions that I can't seem to find solid answers to.  One of the questions is whether or not I can create two forward nodes on the same box without having to set up a virtual machine.  From what I have gathered so far from the Security Onion Documentation pdf (105/307 so far) this will be a 1 box or 1 VM for each node.  Can someone please direct me to where I can confirm this?

Also, as I said, this is my first brush with Security Onion and I have to basically become an expert.  If anyone has a website or book I can buy to help get me on the right path, please let me know.

Thank you all for your time!

**Comment by weslambert** [comment, marked as answer]
Correct, you cannot set up two forward nodes on one physical box without running some sort of hypervisor and provisioning them as separate VMs (we also recommend bare metal only for forward nodes whenever possible).
